finder-app/makefile

```diff
@@ -0,0 +1,11 @@
+.DEFAULT_GOAL := writer
+CROSS_COMPILE = 
+
+writer: writer.o
+	$(CROSS_COMPILE)-gcc writer.o -o writer
```

Comment:
@Ganeshkm10 see notes in the lecture videos and slides about the way to use CROSS_COMPILE, this won't work with future assignments.